examples/caching_and_replay/model.py

```diff
@@ -39,48 +40,64 @@ class Schelling(mesa.Model):
     Model class for the Schelling segregation model.
     """
 
-    def __init__(self, width=20, height=20, density=0.8, minority_pc=0.2, homophily=3):
-        """ """
-        super().__init__()
-        self.width = width
+    def __init__(
+        self,
+        height=20,
+        width=20,
+        homophily=3,
+        radius=1,
+        density=0.8,
+        minority_pc=0.3,
+        seed=None,
+    ):
+        """
+        Create a new Schelling model.
+
+        Args:
+            width, height: Size of the space.
+            density: Initial Chance for a cell to populated
+            minority_pc: Chances for an agent to be in minority class
+            homophily: Minimum number of agents of same class needed to be happy
+            radius: Search radius for checking similarity
+            seed: Seed for Reproducibility
+        """
+
+        super().__init__(seed=seed)
         self.height = height
+        self.width = width
         self.density = density
         self.minority_pc = minority_pc
         self.homophily = homophily
+        self.radius = radius
 
         self.schedule = mesa.time.RandomActivation(self)
-        self.grid = mesa.space.SingleGrid(width, height, torus=True)
+        self.grid = mesa.space.SingleGrid(height, width, torus=True)
```

Comment:
Shouldn't it be width first?

Reply:
I tried to follow the benchmark schelling model, and in that height is first. 

But on checking `mesa.space.SingleGrid` width is indeed first. The change needs to be done in the benchmark code also. I have submitted a PR previously for benchmark, I'll make the change in all the schelling models in this repo and the maine mesa repo